File: devel/libevent/files/patch-buffer.c
#
# Define va_copy if not exists. 4.x for exemple.
#
--- buffer.c.orig	Mon Dec  4 11:34:31 2006
+++ buffer.c	Mon Dec  4 11:35:44 2006
@@ -138,6 +138,9 @@
 		buffer = (char *)buf->buffer + buf->off;
 		space = buf->totallen - buf->misalign - buf->off;
 
+#ifndef va_copy
+#define	va_copy(dest,src)	memcpy(&dest, &src, sizeof (va_list))
+#endif
 		va_copy(aq, ap);
 
 #ifdef WIN32

File: math/octave-forge-signal/files/patch-buffer.m
--- ../inst/buffer.m.orig	2022-04-08 21:56:59.940849000 -0500
+++ ../inst/buffer.m	2022-04-08 21:57:48.381303000 -0500
@@ -120,7 +120,7 @@
           endwhile
           [i, j] = ind2sub([n-p, m], l);
           if (all ([i, j] == [n-p, m]))
-            off --;
+            off--;
           endif
           y (:, end - off + 2 : end) = [];
         else
